.eslint-doc-generatorrc.js

```diff
@@ -19,7 +19,7 @@ const config = {
 		'hasSuggestions',
 		'requiresTypeChecking',
 	],
-	urlConfigs: 'https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs-eslintconfigjs',
+	urlConfigs: 'https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs',
```

Comment:
Should we link to `https://github.com/sindresorhus/eslint-plugin-unicorn#recommended-config` instead?